### DISCUSSION TITLE
New Progress Bar Is Great - How do I read it?

### DISCUSSION THREAD

**Discussion by TSlyCooper** [UltimaHoarder/UltimaScraper#839] [Q&A]
I may be being dense since its late, but how do I read this? Is there a certain resolution it should be to be aligned properly? Sometimes its vertical, diagonal or not lined up at all? Where should I be looking? I think a progress bar is a great idea but I'm clearly not using it correctly.

**Comment by headphonia** [comment]
I don't know about Windows, but in Terminal on macOS, the progress bar is always horizontal. It shows the overall progress (length of the filled section) and process of the file being downloaded (slightly narrower part of the filled section). I think it's great for monitoring the progress of larger video files and it's cleaner than listing every downloaded file for each scrape session.

**Reply by TSlyCooper** [reply]
I played around with the dimensions a bit and it makes sense now. For clarity this is what I was seeing before:

![foto_no_exif](https://user-images.githubusercontent.com/76135562/108637658-92477500-7459-11eb-9f42-93335c2b105f.jpg)

**Reply by headphonia** [reply]
That's quite different from what I see.

<img width="567" alt="Screen Shot 2021-02-22 at 10 10 54 PM" src="https://user-images.githubusercontent.com/77083473/108798081-dff7d680-755a-11eb-8fa0-41ceaa13c04d.png">
